server/data/restClient.ts

```diff
@@ -72,6 +79,14 @@ export default class RestClient {
 
       return raw ? result : result.body
     } catch (error) {
+      if (handle500 && error.response?.status === 500) {
+        const warnings: ErrorSummaryItem[] = []
+        warnings.push({ text: errorMessageFor500 })
+        error.response.errors = warnings
+        logger.info('Handling 500')
+        return error.response
+      }
+      if (handle404 && error.response?.status === 404) return null
```

Comment:
Mind explaining this change? I don't fully understand it, and I'm cautious that it's changing the restClient which is used across multiple different services

Reply:
it allows optinal handling for 404s and 500s coming back from APIs, we have it in MmS. So we can put messages back when an assessment has not been found or delius is down etc. Seems a shame to just bomb out if only one of the apis returns a 404 or 500.